package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "musiquejs",
+  "version": "1.0.0",
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏0.0.1*